### PR TITLE
Added Grype image

### PR DESCRIPTION
**Pull request by mackeyaj**
# PR Details

## Description

This is my first contribution to SDP-Images please let me know if I need to make any changes. Added Grype folder containing Dockerfile used to build the Grype container image, Makefile for running and pushing the build, license and container-entrypoint.sh script

## How Has This Been Tested

This image was tested by building and pushing to the UIP container registry. A grype sdp-library is in development and was used in conjunction with the container image to test the intended functionality of the container, which is to execute a grype scan against a container image.

## Types of Changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] Need assistance with determining if doc updates are needed.

### CODE REVIEW

File: grype/Dockerfile
Comment (psig-bah) [inline grype/Dockerfile:15]: change `version` and `release` to match application/tool version ("0.32.0")

**Review by psig-bah — CHANGES_REQUESTED** [review]
initial review: have a few minor tweaks for the Dockerfile, and I think all of the `syft/` files can be dropped from this PR/branch, since they're covered in #41, correct?

File: grype/Dockerfile
Comment (mackeyaj) [reply grype/Dockerfile]: changed both to 0.32.0

File: grype/Dockerfile
Comment (mackeyaj) [reply grype/Dockerfile]: changed maintainer to uip distro

**Comment by psig-bah** [comment]
almost good to go
@mackeyaj could you merge master back into your PR branch to get the `syft` library files back in sync before I merge this in?

**Comment by va-occ-ess-service-acct** [comment]
Should be good to go.

**Review by psig-bah — APPROVED** [review]

**Pull request merged**
